README.md

```diff
@@ -1,26 +1,65 @@
+| --- | --- | --- | --- | --- |
+| **Steam Store Page** | **Follow Us On Twitter** | **Join The Discord** | **Read The Developer Blog** | **Visit The Website** |
+
+## About 
+
+**Quaver** is a community-driven and open-source competitve vertical scrolling rhythm game with two game modes and online leaderboards. It also features an in-game editor and a flexible multiplayer mode.
+
+It is also being officially released on [Steam](https://store.steampowered.com/app/980610/Quaver/) for Windows, Mac, and Linux - making it one of the most accessible community-driven rhythm games to date.
+
+## Status
+
+This project is currently under heavy development and is in an alpha state, however it is extremely playable as a standalone game. Quaver still has a ton of development ahead of it and will continue to get better as time goes on with new features and bug fixes being added in each day.
+
+Occasionally we open registrations where you can join the Steam alpha and get exclusive access to all the features Quaver has to offer including online rankings, multiplayer, in-game chat, map downloading/uploading, and so much more. Registrations are **currently closed,** however you can still enjoy a limited version of Quaver by compiling the `develop` branch or by downloading one of the [GitHub Releases](https://github.com/Quaver/Quaver/releases).
+
+## Features
+
+Although there are many games like this, **Quaver** offers many unique features and expands on the ideas of its predecessors that make it much different. It is also heavily inspired by a variety of our favorite rhythm games.
+
+* **Two Game Modes** - Play the game with 4 keys or challenge yourself with 7. Each game mode has separate global and country leaderboards to compete on.
+
+* **Global & Country Leaderboards** - Compete with your country and the world for the highest scores and aim to be #1.
+
+* **Ranked & Casual Multiplayer** - Challenge up to 16 players in casual or competitive-oriented multiplayer battles. Multiplayer includes three game modes including Free-For-All, Teams, and Battle Royale.
+
+* **Map Editor** - Create your own maps to any of your favorite songs. Upload and share them with the world, and submit them for official ranking.
+
+* **Advanced Skinning System** - Completely customize your gameplay experience with the ability to create skins. Export and share your skins with friends or upload them to the Steam workshop.
+
+* **Replays** - Go back in time by watching your previous scores. Watch replays from other players around the world, or export your own and share them with your rivals.
+
+* **Over 10+ Game Modifiers** - Switch up the way you play by activating in-game modifiers. Customize the speed of the song, get rid of all the long notes, or even randomize the entire map!
```

Comment:
Well, I wouldn't go as far as calling the lane randomization "randomize the _entire map_"

Reply:
Then what? It's a copy-paste from the Steam page, so it should be updated in both places